vignettes/tcplfit2-vignette.Rmd

```diff
@@ -362,3 +368,263 @@ The output table resulting from the previous code chunk is the same format as th
     concRespPlot(res_wide[i,],ymin=-50,ymax=50)
   }
 ```
+
+## Example 5: How to "Censor" BMD
+The estimated BMD can occurs above or below an experimental dose or other specified values; e.g., it is higher or lower than the maximum or minimum concentration tested in the data. When this happens, `tcplhit2_core` and `concRespCore` provide options for users to "censor" BMD with use of arguments `bmd_low_bnd` and `bmd_up_bnd`. This section will provide examples to demonstrate how to do that. 
+
+`bmd_low_bnd` and `bmd_up_bnd` are multipliers applied to the minimum or maximum concentrations in the data to generate lower or upper bound for BMD estimates. 
```

Comment:
Minor update: "minimum or maximum tested concentrations (i.e. reference doses) to generate the lower and upper boundaries for BMD estimates (i.e. threshold doses), respectively."